eds/blocks/partners-navigation/partners-navigation.js

```diff
@@ -345,10 +345,17 @@ class Gnav {
       if (icon.querySelectorAll('div').length !== 2) {
         return;
       }
-      const iconKey = icon.querySelectorAll('div')[0]?.textContent;
-      if (iconKey.includes(PERSONALIZATION_MARKER)) return;
+
```

Comment:
what is the prupose of this? seem to be not directly related to the ticket, correct?